src/huggingface_hub/file_download.py

```diff
@@ -49,11 +49,13 @@
 
 _tf_version = "N/A"
 _tf_available = False
-try:
-    _tf_version = importlib_metadata.version("tensorflow")
-    _tf_available = True
-except importlib_metadata.PackageNotFoundError:
-    pass
+for package_name in ["tensorflow", "tensorflow-cpu", "tensorflow-gpu"]:
```

Comment:
Might as well rely on `transformers` and copy/paste all the packages we've faced up to now :smile:

https://github.com/huggingface/transformers/blob/fbf468b0573baddb1b9d1bb088a8b6d5c9303a7e/src/transformers/file_utils.py#L83-L93

Reply:
That's a lot of `tensorflow` 😄